drivers/gpu/drm/vc4/vc4_plane.c

```diff
@@ -28,6 +28,12 @@
 #include "vc4_drv.h"
 #include "vc4_regs.h"
 
+#include <linux/module.h>
+#include <linux/moduleparam.h>
+static int siting_h = ~0, siting_v = ~0;
+module_param(siting_h, int, 0644);
+module_param(siting_v, int, 0644);
```

Comment:
Do we really need runtime override of this other than for debugging the original code? Can we just change the default property values instead?

Reply:
I commented this commit should be dropped prior to merge.
It was useful for me (so I could see effect of tweaking values, say on a paused frame in kodi).
I wasn't sure if you can modify plane properties you haven't created through modetest.